src/runtime/virtcontainers/persist/api/config.go

```diff
@@ -185,6 +185,10 @@ type HypervisorConfig struct {
 
 	// TxRateLimiterMaxRate is used to control network I/O outbound bandwidth on VM level.
 	TxRateLimiterMaxRate uint64
+
+	// EPCSize specifies the size in bytes for the EPC Section.
+	// Enable SGX. Hardware-based isolation and memory encryption.
+	EPCSize int64
```

Comment:
I'm thinking would it be better to have SGX in these variable names...

Reply:
SGXEPCSize ?

Reply:
done